From 0dcbe40dc33cc31df85e309a442bd11f76c8843f Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Mon, 17 Oct 2016 15:12:08 +0800
Subject: [PATCH 1237/5242] MLK-13387-2 regulator: consumer: add new event
 macro

commit  8dcc4d5840083639d969499b6a0c7c3faabfe2a2 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add the AFT_ENABLE event macros, because that
1.8v of imx7d pcie phy, should be turned on after
the 1p0d(1.0v) of pcie phy is turned on.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/linux/regulator/consumer.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/regulator/consumer.h b/include/linux/regulator/consumer.h
index 8fe6df70..bb9693be 100644
--- a/include/linux/regulator/consumer.h
+++ b/include/linux/regulator/consumer.h
@@ -123,6 +123,7 @@
 #define REGULATOR_EVENT_ENABLE			0x1000
 #define REGULATOR_EVENT_PRE_DO_ENABLE		0x2000
 #define REGULATOR_EVENT_PRE_DO_DISABLE		0x4000
+#define REGULATOR_EVENT_AFT_DO_ENABLE		0x8000
 
 /*
  * Regulator errors that can be queried using regulator_get_error_flags
-- 
1.7.9.5

